### PR TITLE
Bump CRI-O CI version and update script

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Update the script to use the latest version of CRI-O.
#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by pjbgf — APPROVED** [review]
/lgtm

File: hack/ci/install-cri-o.sh [hunk 1]
```diff
@@ -15,49 +15,22 @@
...
-        /usr/local/bin/runc \
-        /usr/local/bin/crun
-
-    chcon -u system_u -r object_r -t bin_t \
-        /usr/local/bin/conmon \
-        /usr/local/bin/crictl \
-        /usr/local/bin/pinns
-
-    chcon -R -u system_u -r object_r -t bin_t /opt/cni/bin
-
```
Comment (JAORMX) [inline hack/ci/install-cri-o.sh:37]: Why are the `chcon` calls no longer needed?

Comment (saschagrunert) [reply hack/ci/install-cri-o.sh:37]: The script already does it :) https://github.com/cri-o/cri-o/pull/5711

**Comment by saschagrunert** [comment]
/retest

CI is green now. :)

**Comment by saschagrunert** [comment]
@pjbgf PTAL

**Review by pjbgf — APPROVED** [review]
/lgtm

**Pull request merged**